Tools/ardupilotwaf/boards.py

```diff
@@ -57,6 +57,11 @@ def configure_env(self, cfg, env):
         # make easy to override them. Convert back to list before consumption.
         env.DEFINES = {}
 
+        env.prepend_value('INCLUDES', [
+            cfg.srcnode.find_dir('libraries/AP_Common/missing').abspath(),
+            cfg.srcnode.find_dir('build').abspath()
```

Comment:
We actually want `bldnode = cfg.bldnode.make_node(cfg.variant)`, and also use `cfg.write_config_header(os.path.join(cfg.variant, 'header_name.h'))`, so that each configuration header lives in their platform's build directory. Unfortunately, waf doesn't set bldnode properly in configuration contexts :/